projects/cashmere/src/lib/form-field/hc-form-field.component.ts

```diff
@@ -100,4 +103,10 @@ export class HcFormFieldComponent implements AfterContentInit {
                 (!!this._control._errorMessage && this._control._errorMessage.length > 0))
         );
     }
+
+    _shouldShowHint(): boolean {
+        return (
+            this._control._errorState === false
```

Comment:
If there is an error then we hide the hint. Is that correct?

Reply:
That is correct!